### ISSUE TITLE
The nitro gauge is misaligned in vertical splitscreen

### ISSUE THREAD

**Issue by Alayan-stk-2** [supertuxkart/stk-code#5072]
See attached picture

![MisalignedSpeedNitro](https://github.com/supertuxkart/stk-code/assets/25536748/13d82aac-2281-420c-ae03-123d10ed0812)

**Comment by Nomagno** [comment]
Hey! This was merged, but my changes (decoupling X and Y scaling IIRC) resulted in the splitscreen item icon size in the GUI being too small. I talked about this with Alayan in private since he realised after merging, but we didn't put it into an issue. I'm not sure how to fix this, as it requires touching the drawing code to make some subjective adjustments to the item icon portion, though I could try. It's either that or reverting my commit, since the icon size is probably unacceptably small for splitscreen. If anyone could test before/after and give their opinion it'd be huge
